_issues/isnt-hillary-too-sick.md

```diff
@@ -6,34 +6,15 @@ categories: health
+In 2015, Hillary’s personal physician, Dr. Lisa Bardack, released a [statement](https://m.hrc.onl/secretary/10-documents/01-health-financial-records/2015-07-28_Statement_of_Health_-_LBardack.pdf) on her medical history. Bardack concluded:
+>“Mrs. Clinton is a healthy 67-year-old female whose current medical conditions include hypothyroidism and seasonal pollen allergies…. She participates in a healthy lifestyle and has had a full medical evaluation, which reveals no evidence of additional medical issues or cardiovascular disease…. She is in excellent physical condition and fit to serve as President of the United States.”
 
-In 2015, Hillary's personal doctor released this
-[statement](https://m.hrc.onl/secretary/10-documents/01-health-financial-records/2015-07-28_Statement_of_Health_-_LBardack.pdf).
-Several key snippets from this include:
+**Fact.** On September 11, 2016, Hillary, who [had pneumonia,](http://www.theatlantic.com/news/archive/2016/09/hillary-clinton-overheated/499514/) stumbled while leaving an event in New York City. Recovered, she [returned to the campaign trail](http://www.latimes.com/nation/la-na-pol-clinton-campaigning-20160915-snap-story.html) four days later. One of the most famous living journalists in United States history, Dan Rather, put [this into context](https://www.facebook.com/theDanRather/posts/10157417560045716):
 
-* "Mrs. Clinton is a healthy 67-year-old female whose current medical
-conditions include hypothyroidism and seasonal pollen allergies",
-* "She exercises regularly, including yogo, swimming, walking, and
-  weight training"
-* "She participates in a healthy lifestyle..."
+>*But let’s be clear - barring any new developments this remains the story of a candidate working hard on the campaign trail who got a mild case of pneumonia and kept going when she maybe should have rested a bit. Everyone gets sick and until we have more proof of something severe, it remains a storyline to monitor but not belabor.*
 
+Dan Rather has well over 40 years of context for Presidential campaigns: he ought to know what should be considered normal.
 
-On September 11, 2016, Hillary had
-[an incident in New York City](https://www.washingtonpost.com/politics/clinton-falls-ill-during-911-memorial-service-in-new-york/2016/09/11/a52e09c2-7855-11e6-ac8e-cf8e0dd91dc7_story.html). One
-of the most famous living journalists in United States history, Dan
-Rather, put
-[this into context](https://www.facebook.com/theDanRather/posts/10157417560045716):
 
-> But let's be clear - barring any new developments this remains the
-> story of a candidate working hard on the campaign trail who got a
-> mild case of pneumonia and kept going when she maybe should have
-> rested a bit. Everyone gets sick and until we have more proof of
-> something severe, it remains a storyline to monitor but not belabor.
-
-Dan Rather has well over 40 years of context for Presidential
-campaigns: he ought to know what should be considered normal.
```

Comment:
My next question to this would be "then why did she say that she had heat exhaustion rather than pneumonia?" Perhaps a follow up answer to that would be a good addition.